resources/model/OpenStudio.idd

```diff
@@ -24951,47 +24954,51 @@ OS:EvaporativeFluidCooler:SingleSpeed,
        \note provided in the input, will be ignored for the StandardDesignCapacity performance input
        \note method. Also, the standard conditions are for water as a fluid type so this performance input
        \note method can only be used with water as a fluid type (as specified in CondenserLoop object).
-  N5 , \field U-factor Times Area Value at Design Air Flow Rate
+  N6 , \field U-factor Times Area Value at Design Air Flow Rate
        \type real
        \units W/K
        \minimum> 0.0
        \maximum 2100000.0
        \autosizable
        \note Only used for Performance Input Method = UFactorTimesAreaAndDesignWaterFlowRate;
        \note for other Performance Input Methods, this field is ignored.
-  N6 , \field Design Water Flow Rate
+  N7 , \field Design Water Flow Rate
        \type real
        \units m3/s
        \minimum> 0.0
        \autosizable
        \ip-units gal/min
        \note Input value is ignored if fluid cooler Performance Input Method= StandardDesignCapacity.
-  N7 , \field User Specified Design Capacity
+  N8 , \field User Specified Design Capacity
        \type real
        \units W
        \minimum> 0.0
        \note Only used for Performance Input Method = UserSpecifiedDesignCapacity;
        \note for other Performance Input Methods, this field is ignored.
-  N8 , \field Design Entering Water Temperature
+  N9 , \field Design Entering Water Temperature
        \type real
        \units C
        \minimum> 0.0
+       \required-field
+       \autosizable
```

Comment:
Just a side node: IMHO Should probably be `\autocalculatable` given that it just reads mandatory numeric values from Sizing:Plant

* https://github.com/NREL/EnergyPlus/pull/11043#discussion_r2444577623

Reply:
Anyways, this field is also

Only used for Performance Input Method = UserSpecifiedDesignCapacity; for other performance input methods this field is ignored.


Hence why we have had a reset function for it as well